src/backend/distributed/sql/citus--10.2-4--11.0-1.sql

```diff
@@ -13,6 +13,56 @@
 #include "udfs/citus_run_local_command/11.0-1.sql"
 #include "udfs/worker_drop_sequence_dependency/11.0-1.sql"
 
+#include "udfs/get_all_active_transactions/11.0-1.sql"
+#include "udfs/get_global_active_transactions/11.0-1.sql"
+#include "udfs/citus_worker_stat_activity/11.0-1.sql"
+#include "udfs/citus_dist_stat_activity/11.0-1.sql"
+
+SET search_path = 'pg_catalog';
+
+CREATE VIEW citus.citus_lock_waits AS
```

Comment:
why are we re-creating the view? Have we changed anything on the view's definition?

Reply:
It's because we drop a UDF this view depends. I'll put a comment explaining this.

Reply:
Ok but when you drop the function this view depends it drops the view too.

Reply:
Added comment explaining this